Experiments/Experiments.xcodeproj/project.pbxproj

```diff
@@ -270,10 +281,14 @@
 			inputFileListPaths = (
 				"${PODS_ROOT}/Target Support Files/Pods-ExperimentsTests/Pods-ExperimentsTests-frameworks-${CONFIGURATION}-input-files.xcfilelist",
 			);
+			inputPaths = (
+			);
```

Comment:
With the CocoaPods removal, we'll also stop seeing these sections being removed, then added back, then removed...